spikesorters/shellscript.py

```diff
@@ -0,0 +1,178 @@
+import subprocess
+import tempfile
+import shutil
+import signal
+import os
+import time
+from typing import Optional, List, Any
+
+
+class ShellScript():
```

Comment:
Oups sorry. i haven't seen this seen before.
Why there is 2 implementation  of ScriptShell ?

Reply:
That was my mistake. I will delete this one.